src/mod/endpoints/mod_sofia/mod_sofia.c

```diff
@@ -2156,6 +2157,14 @@ static switch_status_t sofia_receive_message(switch_core_session_t *session, swi
 						 TAG_END());
 
 				switch_safe_free(headers);
+
+				if (switch_event_create(&event, SWITCH_EVENT_SEND_INFO) == SWITCH_STATUS_SUCCESS) {
```

Comment:
Can we fire this in nua_r_info event instead?

Reply:
Good point - Since `nua_r_info` is triggered after receiving the 200 OK for the INFO message.

Reply:
Done